transport/dtls/SslDtls.cpp

```diff
@@ -26,6 +26,9 @@ int verify(int, X509_STORE_CTX*)
 
 EVP_PKEY* generateRsaKey()
 {
+#if OPENSSL_VERSION_MAJOR >= 3
+    return EVP_RSA_gen(keySize);
+#else
```

Comment:
I think actually this works already on openssl 1.1.1. The compiler compains about deprecated methods.